modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js

```diff
@@ -352,6 +352,12 @@
 			if (currentElement) {
 				const url = currentElement.getAttribute('href');
 
+				// since we know submitForm(form, 'javascript:;') does nothing
```

Comment:
Hey @NemethNorbert, when I wrote the sample snippet I didn't mean we should commit this comment which will surely not go through review.

Our main agreement for this is to add the LPS number in the comment if we think it's important because otherwise it's hard to understand where the code is coming from.